### DISCUSSION TITLE
Inconsitency when copying streams compared to version 12.3.0

### DISCUSSION THREAD

**Discussion by ljoseph01** [PyAV-Org/PyAV#1673] [4. Bugs]
Hi everyone,

I'm new to this software so please excuse any ignorance. I'm having trouble copying data from one container to another. For now, my goal is simply to duplicate an mp4 file, taking only the video stream. I have this working in 12.3.0 but when running 14.0.0, it doesn't work.

First, my working 12.3.0 code:
```
import av

input_container = av.open(<filename>)
output_container = av.open(<filename>, "w")

for stream in input_container.streams:
    if stream.type == 'video':
        output_stream = output_container.add_stream(template=stream)
        output_stream.time_base = stream.time_base

# Read and mux the packets from the input container
for packet in input_container.demux():
    if packet.stream.type == 'video':
        # Write the video packets to the output container
        packet.stream = output_stream
        output_container.mux(packet)

# Close the containers
input_container.close()
output_container.close()
```

This works perfectly, doing exactly what I want it to do. The output file is an audio-less subtitle-less version of the original. When moving to version 14.0.0, the only change I make is changing `add_stream(template=stream)` to `add_stream_from_template(stream)`, which I believe is intended to be the same thing.

Running the code with `av` version 14.0.0 does yield an output file, however playback is impossible. The video player says that no frames could be decoded within the stream. 

I should mention, the video I am using is an H.265 encoded mp4 file. As far as I can tell, it only contains I frames and P frames (no B frames). Earlier I had some trouble on the encoding side but once I installed `libx265-dev` and reinstalled `ffmpeg`, everything worked fine.

Thanks in advance for assistance.

**Comment by CTHRU** [comment]
I have a similar issue when using `add_stream_from_template()` with an input stream as template. When trying to mux(packet), it complains about a ZERO time base.

If anybody with knowledge knows how to create new video and audio streams with the same parameters as their corresponding input stream, it would be of great help.
